tests/find_package/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.17)
```

Comment:
Pretty cool that we can test find_package!

Reply:
I added testing for `add_subdirectory` usage as well. The both use the same cpp file to ensure there is no difference in how you consume the library.

While we only use find_package for multiple reasons, there are still users who prefer add_subdirectory/FetchContent outside our organization.

Reply:
Ok!